internal/services/events_test.go

```diff
@@ -72,7 +72,7 @@ func TestProcessEvent(t *testing.T) {
 
 func Test_DelegationExpansion(t *testing.T) {
 	// Few important notes:
-	//	- this test uses real data from bsn-devnet, in case of error you can always double-check source of truth
```

Comment:
I think we should leave it as bsn-devnet still exists and one can refer to it, if we decide to switch to another devnet data we will update this comment accordingly